docs/examples/03_backends_ros/files/gh_robot_visualisation.py

```diff
@@ -0,0 +1,47 @@
+"""Visualizes a robot.
+    Inputs:
+        robot: The robot
+        group: str
+            The planning group used for visualisation
+        full_configuration: :class:`compas_fab.robots.Configuration`
+            The robot's full configuration to display.
+        show_visual: bool
+           Whether or not to show the robot's visual meshes.
+        show_collision: bool
+           Whether or not to show the robot's collision meshes.
+        show_frames: bool
+           Whether or not to show the robot's joint frames.
+        show_base_frame: bool
+           Whether or not to show the robot's base frame.
+        show_end_effector_frame: bool
+            Whether or not to show the robot's end-effector frame.
+    Output:
+        visual: The robot's visual meshes.
+        collision: The robot's collision meshes.
+        frames: The robot's joint frames.
+        base_frame_WCF: The robot's base frame.
+        ee_frame_WCF: The robot's end-effector frame.
+"""
+from __future__ import print_function
+
+if robot and full_configuration:
+    robot.update(full_configuration, visual=show_visual, collision=show_collision)
+    print(full_configuration)
+
+    if show_visual:
+        visual = robot.draw_visual()
+
+    if show_collision:
+        collision = robot.draw_collision()
+
+    if show_frames:
+        axes = robot.transformed_axes(full_configuration)
+        frames = robot.transformed_frames(full_configuration)
+
+    if show_base_frame:
+        base_frame_WCF = robot._get_current_base_frame(full_configuration, group)
```

Comment:
actually i totally removed it in bf9db8ca369b52f4f6b6c689e41c38d61c71c5ac, since all planning requests now operate in world frame (see db2fc3bb5dbc96b1a493a4c722406cb6e6efa42b).
But I extended `get_base_frame` with an optional parameter to have the same functionality: 2064ee3f8321db3842350b494902310758178286